### PR TITLE
Update the CI configuration and fix reported issues

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: .travis.yml
Comment (lopuhin) [inline .travis.yml:1]: @Gallaecio sorry I might be out of the loop here, it reads as if we're only testing older python versions and not even 3.8, is this intended?

Comment (Gallaecio) [reply .travis.yml:1]: Oh, forgot to remove that file.

**Review by wRAR — APPROVED** [review]

**Review by wRAR — APPROVED** [review]

File: formasaurus/formtype_features.py [hunk 1]
```diff
@@ -39,27 +40,24 @@ class FormElements(BaseFormFeatureExtractor):
...
     def get_form_features(self, form):
         typecounts = _get_type_counts(form)
         return {
-            'has <textarea>': typecounts['textarea'] > 0,
-            'has <input type=radio>': typecounts['radio'] > 0,
-            'has <select>': typecounts['select'] > 0,
-            'has <input type=checkbox>': typecounts['checkbox'] > 0,
-            'has <input type=email>': typecounts['email'] > 0,
-
-            '2 or 3 inputs': len(form.inputs.keys()) in {2, 3},
```
Comment (lopuhin) [inline formasaurus/formtype_features.py:51]: quite minor but would be great to preserve the grouping, e.g. with adding empty comments between the lines

**Review by lopuhin — APPROVED** [review]
Looks great, thanks @Gallaecio 👍

File: formasaurus/formtype_features.py (hunk 1 above)
Comment (Gallaecio) [reply formasaurus/formtype_features.py:51]: Done. Although I left the comments empty.

Comment (lopuhin) [reply formasaurus/formtype_features.py:51]: Great, that's what I'd do as well

File: README.rst [hunk 2]
```diff
@@ -29,3 +29,9 @@ is a password field and which is a search query, etc.
 License is MIT.
 
 Check `docs <http://formasaurus.readthedocs.org/>`_ for more.
+
+----
+
+.. image:: https://hyperiongray.s3.amazonaws.com/define-hg.svg
+	:target: https://www.hyperiongray.com/?pk_campaign=github&pk_kwd=formasaurus
+	:alt: define hyperiongray
```
Comment (kmike) [inline README.rst]: See https://github.com/zytedata/html-text/pull/11

**Review by kmike — APPROVED** [review]

**Pull request merged**